new-nextjs-client/src/styles/variables.ts

```diff
@@ -27,6 +27,7 @@ const blue2 = '#397BDD';
 /**************** 
   Control Sizes (Input, Button, etc.)
 ****************/
+const controlHeightSM = '3.4rem';
```

Comment:
Nice job on using the variable for this!

Reply:
thank you! 😄